radar/infrastructure_config/identityserver.yml

```diff
@@ -0,0 +1,14 @@
+version: 1
+name: IdentityServer
+blip:
+  - version: "2.6"
+    ring: ADOPT
```

Comment:
Considering future cloud architecture I'm inclined to view this as a `HOLD` technology.

Reply:
Changing to HOLD